front/app/containers/Admin/messaging/texting/PreviewMessage/index.tsx

```diff
@@ -208,12 +242,12 @@ const TextMessagePreview = (props: WithRouterProps) => {
           </Box>
         </Box>
         <Button
-          marginTop="15px"
+          marginTop="25px"
```

Comment:
I think it's best to work in 1 direction (top->bottom, left->right) to avoid margin clashes + hard-to-restyle components. This is something we should perhaps add to our onboarding docs.

Reply:
So would rather have the marginBottom on the component above.

Reply:
It actually is in the onboarding notes here: https://www.notion.so/citizenlab/Onboarding-notes-be63f5bb6bb44e49a536583b225f7727

I just checked these again last night in anticipation of writing down some of my own onboarding notes. I'll fix it now